src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -220,12 +216,11 @@ internal static bool ShowCrashErrorReportWindow(DynamoViewModel viewModel, Crash
                         filesToSend.Add(settingsFile);
                     }
 
-                    if (args.SendDynFile && model != null)
+                    if (args.HasDetails())
                     {
-                        var dynFilePath = Path.Combine(cerDir.FullName, "DynamoModel.dyn");
-                        model.CurrentWorkspace.Save(dynFilePath);
-
-                        filesToSend.Add(dynFilePath);
+                        var stackTracePath = Path.Combine(cerDir.FullName, "StackTrace.log");
+                        File.WriteAllText(stackTracePath, args.Details);
+                        filesToSend.Add(stackTracePath);
```

Comment:
send the call stack as it was recorded in the old crash prompt

Reply:
thanks!

Reply:
Some context: 
@mjkkirschner noticed that the new CER UI cannot show the callstack (like the old prompt). IT was useful to quickly diagnose the problem. To view the callstack in CER, we will need to go to cer.autodesk.com and find the report (and sometimes download the callstack file, if cer.autodesk.com does not display).

not sure if it is a good UX...but we could try to show the callstack like we show the recorded commands (or even in the same document as the commands). 
At the moment the recorded commands (xml) will be opened in a separate process when a crash occurs.

@mjkkirschner @QilongTang any opinions?

Reply:
I think as long as where to find the files gets documented somewhere it's not necessary to popup another window during a crash.